interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+
+            <table class="flowboard" cellpadding='5' cellspacing='2' id="ds_report">
+                <tr class="head">
+
+                    <td><?php xl('Date', 'e'); ?></td>
+                    <td><?php xl('Facility', 'e'); ?></td>
+                    <td><?php xl('Provider', 'e'); ?></td>
+                    <td><?php xl('Appointments', 'e'); ?></td>
+                    <td><?php xl('New Patients', 'e'); ?></td>
+                    <td><?php xl('Visited Patients', 'e'); ?></td>
+                    <td><?php xl('Total Charges', 'e'); ?></td>
+                    <td><?php xl('Total Co-Pay', 'e'); ?></td>
+                    <td><?php xl('Balance Payment', 'e'); ?></td>
+                </tr>
+                <?php
+                if (count($dailySummaryReport) > 0) { // check if daily summary array has value
+                    foreach ($dailySummaryReport as $date => $dataValue) { //   daily summary array which consists different/dynamic values
+                        foreach ($facilities as $facility) { // facility array 
+                            if (isset($dataValue[$facility])) {
+                                foreach ($dataValue[$facility] as $provider => $information) { // array which consists different/dynamic values
+                                    ?>
+                                    <tr>
+                                        <td><?php echo $date ?></td>
+                                        <td><?php echo $facility; ?></td>
+                                        <td><?php echo $provider; ?></td>
+                                        <td><?php echo isset($information['appointments']) ? $information['appointments'] : 0; ?></td>
+                                        <td><?php echo isset($information['newPatient']) ? $information['newPatient'] : 0; ?></td>
+                                        <td><?php echo isset($information['visits']) ? $information['visits'] : 0; ?></td>
+                                        <td align="right"><?php echo isset($information['payments']) ? number_format($information['payments'], 2) : number_format(0, 2); ?></td>
+                                        <td align="right"><?php echo isset($information['paidAmount']) ? number_format($information['paidAmount'], 2) : number_format(0, 2); ?></td>
+                                        <td align="right">
+                                            <?php
+                                            if (isset($information['payments']) || isset($information['paidAmount'])) {
+                                                $dueAmount = number_format(str_replace(",", "", $information['payments']) - str_replace(",", "", $information['paidAmount']), 2);
+                                            } else {
+                                                $dueAmount = number_format(0, 2);
+                                            }
+                                            echo $dueAmount;
```

Comment:
We have used text() function as per your above comments.
Please see the changes at pull request https://github.com/bradymiller/openemr/pull/9